<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionDialog</class>
 <widget class="QDialog" name="ConnectionDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>192</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Connection Ã  softOsteo</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="spacing">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="UsernameLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="UsernameLabel">
       <property name="text">
        <string>Nom d'utilisateur:</string>
       </property>
       <property name="buddy">
        <cstring>UsernameTexte</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="UsernameTexte"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="PasswordLayout">
     <item>
      <widget class="QLabel" name="PasswordLabel">
       <property name="text">
        <string>Mot de passe:</string>
       </property>
       <property name="buddy">
        <cstring>PasswordTexte</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="PasswordTexte"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="QuitButton">
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OkButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>UsernameTexte</tabstop>
  <tabstop>PasswordTexte</tabstop>
  <tabstop>OkButton</tabstop>
  <tabstop>QuitButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
